src/LiveComponent/src/Test/InteractsWithLiveComponents.php

```diff
@@ -44,4 +44,24 @@ protected function createLiveComponent(string $name, array $data = [], ?KernelBr
             self::getContainer()->get('router'),
         );
     }
+
+    protected function assertComponentEmitEvent(TestLiveComponent $testLiveComponent, string $eventName, ?array $parameters = null): void
+    {
+        $eventData = $testLiveComponent->getEmittedEvent($testLiveComponent->render(), $eventName);
+
+        $this->assertNotNull($eventData, \sprintf('Event %s not emitted', $eventName));
```

Comment:
Can we have something like that instead?

```suggestion
        $this->assertNotNull($event, \sprintf('The component "%s" did not emit event "%s".', '...', $eventName));
```